faqs/galaxy/workflows_variables.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Rename outputs in a workflow using workflow variables
+area: workflows
+box_type: tip
+layout: faq
+contributors: [delphine-l]
+---
+
+Workflow text inputs can be used as parameters for tools but also in the workflow editor for renaming files. 
+
+# Create a workflow variable
+
+1. Open the workflow editor
+2. Click on **Inputs** in the toolbar on the left
+3. Click on **Simple inputs used for Workflow logic**
+4. Click on the created input box in the workflow editor
+5. Label your input. Example: `Species Name`
```

Comment:
I guess it would be useful to overwrite this example name from within tutorials.

See for example https://raw.githubusercontent.com/galaxyproject/training-material/a8c16060c1a15f5109d46a10ca36d0e9dbad1597/faqs/galaxy/datasets_import_via_link.md